conda/e3sm_diags_env_dev.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
   - _openmp_mutex=4.5
   - attrs=20.3.0
   - beautifulsoup4=4.9.3
+  - black=20.8b1
```

Comment:
Will any of these dependency additions cause issues for the unified environment?

Reply:
Adding dependencies to development env shouldn't impact unified env. We may don't need these additions in e3sm_diags_env_osx.yml (non-development env) though.